Sources/Extensions/NSAttributedStringExtension.swift

```diff
@@ -9,15 +9,15 @@
 import Cocoa
 
 public extension NSAttributedString {
-    public class func attributedStringFromHTML(html: String, withCSSString css: String?) -> NSAttributedString {
+    public class func attributedStringFromHTML(_ html: String, withCSSString css: String?) -> NSAttributedString {
```

Comment:
This extension is used by the Sample. Maybe it should be in the sample project because users of SwiftMark may not have use of this extension.